### PR TITLE
Add community

### PR DESCRIPTION
**Pull request by omerltf**

### CODE REVIEW

File: CommunityShed/CommunityShed/PersonEdit.aspx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+            <div class="form-group col p-3">
+                <asp:Label ID="EmailLabel" Text="Email: " runat="server" AssociatedControlID="EmailTextBox" />
+                <asp:TextBox ID="EmailTextBox" CssClass="form-control" runat="server" />
+            </div>
+
+            <div class="form-group col p-3">
+                <asp:Label ID="PasswordLabel" Text="Password: " runat="server" AssociatedControlID="PasswordTextBox" />
+                <asp:TextBox ID="PasswordTextBox" CssClass="form-control" TextMode="Password" runat="server" />
+            </div>
```
Comment (appacademy-jchurchill) [inline CommunityShed/CommunityShed/PersonEdit.aspx]: Remove the password field.

**Review by appacademy-jchurchill — CHANGES_REQUESTED** [review]
Let's remove the user password field on the edit page.

File: CommunityShed/CommunityShed/PersonEdit.aspx (hunk 1 above)
Comment (omerltf) [reply CommunityShed/CommunityShed/PersonEdit.aspx]: Done. Please review

Comment (appacademy-jchurchill) [reply CommunityShed/CommunityShed/PersonEdit.aspx]: Looks good.

**Comment by appacademy-jchurchill** [comment]
Let's merge this. :shipit:

**Review by appacademy-jchurchill — APPROVED** [review]

**Pull request merged**